src/test/resources/projects/clientserver/server/build.gradle

```diff
@@ -24,7 +24,7 @@ sourceCompatibility = 1.7
 targetCompatibility = 1.7
 
 dependencies {
-  compile "com.google.endpoints:endpoints-framework:2.0.7" // use latest
+  compile "com.google.endpoints:endpoints-framework:2.0.9" // use latest
   compile "com.google.appengine:appengine-api-1.0-sdk:+" // use latest
```

Comment:
On the other hand I did write this line with the appengine-api-1.0-sdk, so I guess I don't know what I'm talking about.